README.md

```diff
@@ -1,207 +1,350 @@
-| Name                                   | Description                           | Argument | Default              |
-|----------------------------------------|---------------------------------------|----------|----------------------|
-| --validator-wallet-dir value           | location of generated wallet          | Path     | "./mainnet-keystore" |
-| --validator-keys-dir value             | Path to your validator keys           | Path     |                      |
-| --validator-wallet-password-file value | Path to your password file            | Path     |                      |
-| --mainnet                              | Run for mainnetFlag (default network) | Bool     | false                |
-| --testnet                              | Run for testnetFlag                   | Bool     | false                |
-| --devnet                               | Run for devnet                        | Bool     | false                |
+As a Genesis Validator you can provide an indicative voting for the prefered initial token supply of LYX, which will determine how much the Foundation will receive. See the https://deposit.mainnet.lukso.network website for details.
+You can choose between:
+1: 35M LYX
+2: 42M LYX (This option is the prefered one by the Foundation)
+3: 100M LYX
+4: No vote
+
+The private key you are using requires enough the LYXe to fund pay for validator keys deposits.
+Make sure your privatekey has sufficient balances:
+320 LYXe
+1.6 ETH for GAS
+Enter private key:
+
+> \_
+
+| Flag                             | Description                                                                                                                 |
+| -------------------------------- | --------------------------------------------------------------------------------------------------------------------------- |
+| --mainnet                        | runs a validator on LUKSO's mainnet (default)                                                                               |
+| --testnet                        | runs a validator on LUKSO's testnet                                                                                         |
+| --devnet                         | runs a validator on LUKSO's devnet                                                                                          |
+| --geth-\*                        | \* Pass any flag to the Geth node [See docs for details](https://geth.ethereum.org/docs/fundamentals/command-line-options)  |
+| --erigon-\*                      | \* Pass any flag to the Erigon node [See docs for details](https://github.com/ledgerwatch/erigon)                           |
+| --prysm-\*                       | \* Pass any flag to the Prysm node [See docs for details](https://docs.prylabs.network/docs/prysm-usage/parameters)         |
+| --lighthouse-\*                  | \* Pass any flag to the Lighthouse node [See docs for details](https://lighthouse-book.sigmaprime.io/advanced-datadir.html) |
```

Comment:
We don't actually use any network specific flag during validator deposit, as well as network flags  - you can delete those and leave those that are below `--lighthouse`. We could also split this section into `deposit` and `init` one, because I can't find any reference to initializing validator